dev-python/pyaudio/pyaudio-0.2.11-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1
+
+MY_PN="PyAudio"
+
+DESCRIPTION="Python bindings for PortAudio"
+HOMEPAGE="http://people.csail.mit.edu/hubert/pyaudio/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_PN}-${PV}.tar.gz"
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RDEPEND="media-libs/portaudio"
+DEPEND="${RDEPEND}
+	test? ( dev-python/numpy )"
+
+distutils_enable_sphinx sphinx
+
```

Comment:
you should add `disutils_enable_tests unittest` as you are using the unittest module.

Reply:
Done